repose-aggregator/components/filters/valkyrie-authorization/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -75,7 +76,15 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
     val tracingHeader = nullOrWhitespace(Option(mutableHttpRequest.getHeader(CommonHttpHeader.TRACE_GUID.toString)))
     val urlPath: String = new URL(mutableHttpRequest.getRequestURL.toString).getPath
     val matchingResources: Seq[Resource] = Option(configuration.getCollectionResources)
-      .map(_.getResource.asScala.filter(_.getPathRegex.r.pattern.matcher(urlPath).matches())).getOrElse(Seq.empty[Resource])
+      .map(_.getResource.asScala.filter(resource => {
+        val pathRegex = resource.getPathRegex
+        pathRegex.getValue.r.pattern.matcher(urlPath).matches() && {
+          val httpMethods = pathRegex.getHttpMethods
+          httpMethods.isEmpty ||
```

Comment:
It appears the reason the default was missing is actually a JAXB/XJC bug.
* http://stackoverflow.com/questions/27528698/is-there-any-way-i-can-specify-defaultvalue-for-collections-with-jaxb